pyfixest/estimation/multcomp.py

```diff
@@ -142,34 +150,48 @@ def rwolf(
     for i in range(S):
         model = models[i]
 
-        wildboot_res_df, bootstrapped_t_stats = model.wildboottest(
-            param=param,
-            reps=reps,
-            return_bootstrapped_t_stats=True,
-            seed=seed,  # all S iterations require the same bootstrap samples, hence seed needs to be reset
-        )
-        t_stats[i] = wildboot_res_df["t value"]
-        boot_t_stats[:, i] = bootstrapped_t_stats
+        if sampling_method == "wild-bootstrap":
+            wildboot_res_df, bootstrapped_t_stats = model.wildboottest(
+                param=param,
+                reps=reps,
+                return_bootstrapped_t_stats=True,
+                seed=seed,  # all S iterations require the same bootstrap samples, hence seed needs to be reset
+            )
+
+            t_stats[i] = wildboot_res_df["t value"]
+            boot_t_stats[:, i] = bootstrapped_t_stats
+
+        elif sampling_method == "ri":
+            model.ritest(
+                resampvar=param,
+                reps=reps,
+                type="randomization-t",
+                store_ritest_statistics=True,
+            )
+
+            t_stats[i] = model._ritest_sample_stat
+            boot_t_stats[:, i] = model._ritest_statistics
+        else:
```

Comment:
Could you add a test to [test_errors.py](https://github.com/py-econometrics/pyfixest/blob/0c84f012e2cf72762893ecf121e447e306b54e10/tests/test_errors.py#L181) that triggers the ValueError? This will make the codecov tool happy =)

Reply:
Could you elaborate on this? In `test_error.py`, we have two functions related to `multcomp.py` : `test_rwolf_error` and `test_multcomp_errors`. `test_multcomp_errors` also triggers a ValueError, but are we adding a new test that triggers the error in the sampling method feature? I would appreciate it if you let me know whether the following code will be enough. Thanks!

```
def test_multcomp_sampling_errors():
    data = get_data().dropna()
    # Sampling method not supported in "rwolf"
    fit1 = feols("Y + Y2 ~ X1 | f1", data=data)
    with pytest.raises(ValueError):
        rwolf(fit1.to_list(), param="X1", reps=999, seed=92, sampling_method = "abc")
```

Reply:
Yes, that's exactly what I had I mind =)